packages/adapter/src/utils.ts

```diff
@@ -51,7 +51,7 @@ export const metaMaskFlaskSupported = async (externalProvider?: any): Promise<bo
   return true
 }
 
-export const shapeShiftSnapInstalled = async (snapId: string): Promise<boolean> => {
+export const shapeShiftMultichainInstalled = async (snapId: string): Promise<boolean> => {
```

Comment:
This is probably some diffs you had locally which explains why hdwallet implementation was hallucinating methods - since there is already a PR that's a few fixes away to make CI happy and it being opened https://github.com/shapeshift/hdwallet/pull/628 can we pls revert

Reply:
I'd rather go ahead and make the proper changes on the hdwallet side as this is the correct method nomenclature given the updated package name.

Reply:
Let's keep diffs clean, avoid friction and keep that for a follow-up if you want to update the names. We currently have a working hdwallet implementation depending on this, and web depending on hdwallet. If the hdwallet PR was made while these methods already existed / were already renamed from the get-go, I'd have no issue with this but the goal right now is to ship

Reply:
5270461df8d5125867707dd9d582f3c7a6ce5aae